### ISSUE TITLE
Missing commas in setup_build.py

### ISSUE THREAD

**Issue by pacohope** [h5py/h5py#882]
Current version of setup_build.py has the following 2 lines (lines 52, 53):
```
    COMPILER_SETTINGS['include_dirs'].extend(['/opt/local/include''/usr/local/include'])
    COMPILER_SETTINGS['library_dirs'].extend(['/opt/local/include''/usr/local/include'])
```
There's supposed to be a comma in there, like so:
```
    COMPILER_SETTINGS['include_dirs'].extend(['/opt/local/include','/usr/local/include'])
    COMPILER_SETTINGS['library_dirs'].extend(['/opt/local/include','/usr/local/include'])
```

I couldn't compile because I was seeing compile lines like this:
```
cc -fno-strict-aliasing -O2 -pipe -fstack-protector -fno-strict-aliasing -DNDEBUG -fPIC -DH5_USE_16_API -I/usr/home/paco/h5py-2.7.0/lzf -I/opt/local/include/usr/local/include -I/usr/home/paco/h5py-2.7.0/.eggs/numpy-1.12.1-py2.7-freebsd-10.3-RELEASE-p18-amd64.egg/numpy/core/include .... etc....
```

Note the `/opt/local/include/usr/local/include` on the command line. If that had been `-I/opt/local/include -I/user/local/include` the way it was meant to be, my stuff would have compiled fine. Surprised this works for anyone.

**Comment by tacaswell** [comment]
That is a fun mistake.  I assume it works for if the required includes and libraries are in the default search paths.

Can you put in a PR to fix that?

**Comment by pacohope** [comment]
Fine. That's a lot of work for 2 commas. :) https://github.com/h5py/h5py/pull/883

**Comment by aragilar** [comment]
Closed by #883
